<launch>
  <param name="/shuttleFinder/offset_x" value="0.0" />
  <param name="/shuttleFinder/offset_y" value="-0.383" />
  <param name="/shuttleFinder/offset_z" value="1.415" />
  <node name="shuttleFinder" pkg="shuttle_finder" type="shuttle_finder" />
</launch>
